content/docs/user-guide/assets/scene-pipeline/procedural_prefab.md

```diff
@@ -0,0 +1,135 @@
+description: A feature to create scipt driven prefab assets
+weight: 400
+toc: true
+---
+
+The procedural prefab feature allows technical pipeline developers to write out O3DE native entity-component scene descriptions in the form of prefab assets. The developers can write Python scripts to parse imported source scenes so that they can write out any number of prefab assets. Before this feature was added to O3DE, each mesh data node would need to be broken out inside the O3DE Editor. The procedural prefab workflow allows the artist to work longer in their digital content creation (DCC) tool such as Maya and Blender.
+
+The workflow produces procedural prefabs using Python scripts inside the scene pipeline or when when the Prefab gem generates a default procedural prefab.
+
+![High level view of how procedural prefabs work with Python scripts](/images/user-guide/assets/scene-pipeline/proc_prefab_workflow.png)
+
+## The characteristics of a procedural prefab
+
+A prefab is a prefabricated description of entities, components, and properties for a scene. Prefabs serve the purpose of easily building and modifying large and complex worlds for games. A user can author a prefab inside the O3DE Editor.
+
+More information about the [Prefab](docs/user-guide/interactivity/prefabs) feature.
+
+It is also possible to define a procedure to produce a product prefab asset. Python scripts are the most typical way to define procedural prefab product assets. This feature give the ability to maintain the prefab template inside a source art scene file such as FBX or glTF. The O3DE content creators are free to instantiate these procedural prefabs in a level and are automatically updated when the source scene is updated.
+
+These procedural prefab assets are treated as read only prefabs inside the Editor since they are created by the definition from the original source scene file plus the script that produced the prefab. In fact, procedural prefabs are placed in the asset cache so they should not be changed directly.
+
+## Producing procedural prefabs
+
+The Prefab gem needs to be enabled by the project to gain the procedural prefab feature. Once the gem is enable then all source scene files will start producing a default procedural prefab and the Python scripts will have the ability to produce prefabs.
+
+{{< note >}}
+It is possible to turn off the default procedural prefab by setting the registry key "/O3DE/Preferences/Prefabs/UseProceduralPrefabs" to False.
+{{< /note >}}
+
+### Creating procedural prefabs from Python
+
+It is possible to create a procedural prefab using a Python script to hook into the scene pipeline. This script will add a Prefab Group rule into the scene manifest that describes the entities and components it will use. In fact, the script can add many Prefab Groups to the scene manifest if there is more than one scene description. The scripts can split up complex scenes from source files and to fill out entities with game components.
```

Comment:
```suggestion
It is possible to create a procedural prefab using a Python script to hook into the scene pipeline. This script will add a Prefab Group rule into the scene manifest that describes the entities and components it will use. In fact, the script can add many Prefab Groups to the scene manifest if there is more than one scene description. The scripts can split up complex scenes from source files and fill out entities with game components.
```